contracts/erc20-bridge-sampler/test/erc20-bridge-sampler.ts

```diff
@@ -716,6 +718,93 @@ blockchainTests('erc20-bridge-sampler', env => {
         });
     });
 
+    blockchainTests.resets('getLiquidityProviderFromRegistry', () => {
+        const xAsset = randomAddress();
+        const yAsset = randomAddress();
+        const sampleAmounts = getSampleAmounts(yAsset);
+        let liquidityProvider: DummyLiquidityProviderContract;
+        let registryContract: DummyLiquidityProviderRegistryContract;
+
+        before(async () => {
+            await testContract.createTokenExchanges([MAKER_TOKEN, TAKER_TOKEN]).awaitTransactionSuccessAsync()
+                .txHashPromise;
```

Comment:
No need to call `createTokenExchanges`. It just creates uniswap exchanges, which you aren't using.